bin/generate-config

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+SERVICE_NAME=${SERVICE_NAME:-nginx}
+CONSUL=${CONSUL:-consul}
+CERT_DIR="/var/www/ssl"
+
+# Determine whether ACME SSL certificates have been acquired
+SSL_READY="false"
+if [ -f ${CERT_DIR}/fullchain.pem -a -f ${CERT_DIR}/privkey.pem ]; then
+    export SSL_READY="true"
+fi
+
+# Generate a conf.d config file for every corresponding cond.d Consul template
+for f in $(ls -1 /etc/nginx/templates/conf.d/)
```

Comment:
Not sure if you missed it in the [consul-template docs](https://github.com/hashicorp/consul-template) but we can render multiple files from a single invocation, which should reduce the number of API calls to Consul. Their example is as follows:

```
$ consul-template \
  -consul my.consul.internal:6124 \
  -template "/tmp/nginx.ctmpl:/var/nginx/nginx.conf:service nginx restart" \
  -template "/tmp/redis.ctmpl:/var/redis/redis.conf:service redis restart" \
  -template "/tmp/haproxy.ctmpl:/var/haproxy/haproxy.conf"
```